pkg/util/utils.go

```diff
@@ -163,3 +164,39 @@ func MaskProxyPasswordWithKey(v string) string {
 	}
 	return v
 }
+
+// remove the specified line from the given file
+func RemoveLineFromFile(knownHostLine string, filePath string) error {
```

Comment:
@ComradeProgrammer hi I think we can add a great unit test for this function

for example we can have a dummy knownhost file in Unit tests, simmilar to a real knownhost with duplicates and then expect the unit test to see it corrected, can we plz add this unit test?